src/app/modules/customer-managment/components/projects-type/components/create-project-type/create-project-type.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <form style="width: 600px;">
+    <div class="form-group">
+      <input type="text" class="form-control form-control-sm" id="" aria-describedby="" placeholder="Name" />
+      <textarea
+        class="form-control form-control-sm mt-2"
+        id="exampleFormControlTextarea1"
```

Comment:
id="exampleFormControlTextarea1" ???
Why not something more meaningful like:
id="descriptionTextArea"